### PR TITLE
fix(TBD-11046): Kafka spark big data streaming doesn't work well(missing dependencies)

### PR DESCRIPTION
**Pull request by sponomarova**
…sing dependencies in hdp 2.x) (#1564)

**Please check if the PR fulfills these requirements**

- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [ ] Unit tests for the Java changes have been added (for bug fixes / features) ?
- [ ] TUJ for the JavaJet changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The new code does not introduce new technical issues

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**BREAKING CHANGE**

If this PR contains a breaking change, please describe the impact and migration
path for existing applications.
If not please remove this section.

**Other information**:

### CODE REVIEW

**Review by lbourgeois — APPROVED** [review]
LGTM

**Review by AlixMetivier — APPROVED** [review]

File: main/plugins/org.talend.hadoop.distribution.hdpx/resources/builtin/hdp2xx/Hortonworks_HDP_2_6_0_3_8.json [hunk 1]
```diff
@@ -5542,7 +5542,17 @@
       "id" : "DYNAMIC_HDP_2_6_0_3_8_SPARK_KAFKA_ASSEMBLY_LIB_MRREQUIRED_DYNAMIC",
       "tagName" : "libraryNeededGroup",
       "templateId" : "SPARK-KAFKA-ASSEMBLY-LIB-MRREQUIRED-DYNAMIC"
-    }, {
+      }, {
+      "childNodes" : [ {
```
Comment (AlixMetivier) [inline main/plugins/org.talend.hadoop.distribution.hdpx/resources/builtin/hdp2xx/Hortonworks_HDP_2_6_0_3_8.json:5546]: where is the [ closed at ?

**Review by AlixMetivier — CHANGES_REQUESTED** [review]

File: main/plugins/org.talend.hadoop.distribution.hdpx/resources/builtin/hdp2xx/Hortonworks_HDP_2_6_0_3_8.json [hunk 2]
```diff
@@ -5542,7 +5542,17 @@
...
+        "childNodes" : [ ],
+        "id" : "DYNAMIC_HDP_2_6_0_3_8_kafka_2_11_0_10_0_1_jar",
+        "tagName" : "library"
+      }, {
+      "childNodes" : [ {
+        "childNodes" : [ ],
+        "id" : "DYNAMIC_HDP_2_6_0_3_8_spark_streaming_kafka_0_10_2_11_2_1_0_2_6_0_3_8_jar",
+        "tagName" : "library"
+      }, {
```
Comment (AlixMetivier) [inline main/plugins/org.talend.hadoop.distribution.hdpx/resources/builtin/hdp2xx/Hortonworks_HDP_2_6_0_3_8.json:5555]:
```
      }, {
      "childNodes" : [ {
        "childNodes" : [ ],
        "id" : "DYNAMIC_HDP_2_6_0_3_8_kafka_2_11_0_10_0_1_jar",
        "tagName" : "library"
      }, {
        "childNodes" : [ ],
        "id" : "DYNAMIC_HDP_2_6_0_3_8_spark_streaming_kafka_0_10_2_11_2_1_0_2_6_0_3_8_jar",
        "tagName" : "library"
      }, {
        "childNodes" : [ ],
        "id" : "DYNAMIC_HDP_2_6_0_3_8_kafka_clients_0_10_1_2_6_0_3_8_jar",
        "tagName" : "library"
      }, {
        "childNodes" : [ ],
        "id" : "DYNAMIC_HDP_2_6_0_3_8_lz4_1_3_0_jar",
        "tagName" : "library"
      }, {
        "childNodes" : [ ],
        "id" : "DYNAMIC_HDP_2_6_0_3_8_slf4j_api_1_7_21_jar",
        "tagName" : "library"
      },
````

Comment (AlixMetivier) [reply main/plugins/org.talend.hadoop.distribution.hdpx/resources/builtin/hdp2xx/Hortonworks_HDP_2_6_0_3_8.json:5555]: @lbourgeois how do you do suggestions, can't make it work :(

File: main/plugins/org.talend.hadoop.distribution.hdpx/resources/builtin/hdp2xx/Hortonworks_HDP_2_6_0_3_8.json [hunk 3]
```diff
@@ -5542,7 +5542,17 @@
...
       "childNodes" : [ {
         "childNodes" : [ ],
```
Comment (AlixMetivier) [inline main/plugins/org.talend.hadoop.distribution.hdpx/resources/builtin/hdp2xx/Hortonworks_HDP_2_6_0_3_8.json:5557]:
```suggestion
      }, {
      "childNodes" : [ {
        "childNodes" : [ ],
        "id" : "DYNAMIC_HDP_2_6_0_3_8_kafka_2_11_0_10_0_1_jar",
        "tagName" : "library"
      }, {
        "childNodes" : [ ],
        "id" : "DYNAMIC_HDP_2_6_0_3_8_spark_streaming_kafka_0_10_2_11_2_1_0_2_6_0_3_8_jar",
        "tagName" : "library"
      }, {
        "childNodes" : [ ],
```

**Review by AlixMetivier — APPROVED** [review]

**Pull request merged**